client/tests/end2end/test-globaleaks-process.js

```diff
@@ -277,6 +277,20 @@ describe('globaLeaks process', function() {
     });
   });
 
+  it('Recipient should be able to export the first submission from the tips page', function(done) {
+    login_receiver(receiver_username, receiver_password).then(function() {
+      element(by.css('#tip-0 form.tipExport button')).click().then(function() {
+        if (utils.testFileDownload()) {
```

Comment:
this line ```if (utils.testFileDownload()) {``` should be swapped with the previous ```element(by.css('#tip-0 form.tipExport button')).click().then(function() {``` in order to avoid the tests on the browsers that requires user input for the file download.

going to do this while merging.